core/kazoo_number_manager/src/providers/knm_failover.erl

```diff
@@ -63,16 +60,12 @@ feature(Number) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec update_failover(knm_number:knm_number()) -> knm_number:knm_number().
-update_failover(Number) ->
-    CurrentFailover = feature(Number),
-    PhoneNumber = knm_number:phone_number(Number),
-    Failover = kz_json:get_ne_value(?KEY, knm_phone_number:doc(PhoneNumber)),
-    NotChanged = kz_json:are_equal(CurrentFailover, Failover),
-    case kz_term:is_empty(Failover) of
-        'true' ->
-            knm_services:deactivate_feature(Number, ?KEY);
-        'false' when NotChanged ->
-            Number;
-        'false' ->
-            knm_services:activate_feature(Number, {?KEY, Failover})
+update_failover(N) ->
+    Private = feature(N),
+    Public = kz_json:get_ne_value(?KEY, knm_phone_number:doc(knm_number:phone_number(N))),
+    NotChanged = kz_json:are_equal(Private, Public),
+    case kz_term:is_empty(Public) of
+        true -> knm_services:deactivate_feature(N, ?KEY);
```

Comment:
Atoms are still to be tickied, changing them here is not helpful in that regard. Please revert